vector/src/test/java/im/vector/app/features/settings/devices/v2/overview/SessionOverviewViewModelTest.kt

```diff
@@ -125,7 +132,7 @@ class SessionOverviewViewModelTest {
                 deviceId = A_SESSION_ID_1,
                 deviceInfo = Success(deviceFullInfo),
                 isCurrentSessionTrusted = true,
-                pushers = Loading(),
+                notificationsEnabled = true,
```

Comment:
I am wondering how it can be `true` in the `ViewState` as it is `false` by default and I don't see any mock of the pushers or account data event for the given SessionId in several tests. It should normally fail.

Reply:
I'm not entirely sure but it looks like the account data event is mapping a default local notification event content even when it doesn't receive such an event. I tried fixing this but because the mapping to content is non-null it doesn't seem so straightforward. It doesn't break anything though, and I would like to revisit this separately as we are on a tight deadline

Reply:
Okay let's refactor if needed in another PR then.